### ISSUE TITLE
[feature] improve the ease with which the search field can be added as a search engine in the browser (firefox)

### ISSUE THREAD

**Issue by Jiehong** [darekkay/static-marks#30]
Hi,

Today, the documentation says that the bookmarks can be used as a search engine by using https://darekkay.com/static-marks/demo/default.html?search=%s.

Which is great (really!)

However, when right-clicking on the search field, the browser does not seem to allow you to do that automatically, even though it should (here, with Firefox 96).

There is a way to fix that easily, though: by converting the `<div class="filter" role="search">` to a form, and giving a name to the input field:

```
<form class="filter" role="search">
  <div class="filter-button empty">
    <button type="button" class="btn-unstyled" aria-label="Clear filter" disabled="">
      <div class="icon icon-search"></div>
    </button>
  </div>
  <input aria-label="Search everywhere" type="search" class="input" placeholder="Search everywhere..." name="search">
</form>
```

(No visual change for the user).

With this, (in Firefox), you can right-click the search field, and select "Add a keyword for this search", and it will smartly understand that you want to use `https://darekkay.com/static-marks/demo/default.html?search=%s` as the search term.

This feature has been removed in Chrome for a few years already, so this won't help there.

**Comment by darekkay** [comment]
Thanks @Jiehong , that's a small but great improvement 🚀  I've applied your suggestion and it works like a charm. The feature will be available in the next release.

**Comment by darekkay** [comment]
Released in [2.4.0](https://github.com/darekkay/static-marks/releases/tag/v2.4.0)
